libs/ui/src/admin/CollectionEdit/CollectionEditForm.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { FC, useEffect, useState } from 'react'
+
+import {
+  Stack,
+  Textarea,
+  ButtonGroup,
+  Button,
+  HStack,
+  useDisclosure,
+} from '@chakra-ui/react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { UploadFile } from '@wsvvrijheid/types'
+import {
+  useDeleteCollection,
+  useUnpublishModel,
+  useUpdateCollectionMutation,
+  usePublishModel,
+} from '@wsvvrijheid/utils'
+import { useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { TFunction } from 'react-i18next'
+import { AiOutlineEdit } from 'react-icons/ai'
+import { BsTrash } from 'react-icons/bs'
+import {
+  MdOutlineUnpublished,
+  MdClose,
+  MdOutlineCheck,
+  MdOutlinePublishedWithChanges,
+} from 'react-icons/md'
+import * as yup from 'yup'
+
+import { FilePicker, FormItem } from '../../components'
+import { WImage } from '../../components'
+import { WConfirm, WConfirmProps } from '../../components/WConfirm'
+import { CollectionEditFormProps, CollectionEditFormFieldValues } from './types'
+
+const schema = (t: TFunction) =>
+  yup.object({
+    title: yup.string().required(t`art.create.form.title-required`),
+    description: yup.string().required(t`art.create.form.description-required`),
+  })
+
+export const CollectionEditForm: FC<CollectionEditFormProps> = ({
+  collection,
+  isEdit,
+  toogleEdit,
+  queryKey,
+}) => {
+  const id = collection.id
+  const isPublish = collection.publishedAt
+  const { t } = useTranslation()
+
+  const [images, setImages] = useState<Blob[]>([])
+  const updateCollectionMutation = useUpdateCollectionMutation(queryKey)
+  const unpublishCollectionMutation = useUnpublishModel(
+    'api/collections',
+    queryKey,
+  )
+  const publishCollectionMutation = usePublishModel('api/collections', queryKey)
+  const deleteCollectionMutation = useDeleteCollection(queryKey)
+
+  const confirmDisclosure = useDisclosure()
+  const [confirmState, setConfirmState] =
+    useState<Omit<WConfirmProps, 'onClose' | 'isOpen' | 'onOpen'>>()
+
+  const {
+    register,
+    formState: { errors, isValid },
+    handleSubmit,
+    reset: resetForm,
+  } = useForm<CollectionEditFormFieldValues>({
+    resolver: yupResolver(schema(t)),
+    mode: 'all',
+    defaultValues: {
+      title: collection.title,
+      description: collection.description,
+    },
+  })
+  const onSaveCollection = async (data: CollectionEditFormFieldValues) => {
+    updateCollectionMutation.mutate({
+      id,
+      ...(images.length > 1 && {
+        image: images[0],
+      }),
+      ...data,
+    })
```

Comment:
You can toggle edit mode after saving the form.

```suggestion
    }, {
      onSuccess: toggleEdit
    })
```